amm/repl/src/test/scala/ammonite/session/BuiltinTests.scala

```diff
@@ -143,7 +143,7 @@ object BuiltinTests extends TestSuite {
       // In 2.12.13, I would have expected things like
       //   interp.configureCompiler(_.settings.Wconf.tryToSet(List("any:wv", "cat=unchecked:ws")))
       // to re-instate the expected warning below, to no avail :|
-      if (TestUtils.scala2_12 && sv.stripPrefix("2.12.").toInt <= 12) {
+      if (TestUtils.scala2_12 && sv.stripPrefix("2.12.").toInt <= 19) {
```

Comment:
> Do you know or have an idea why the test isn't configured for Scala 2.13? Also newer Scala 2.12 version aren't covered?

@lefou after covering until `2.12.19` this was successful: `mill 'amm.repl[2.12.{13,14,15,16,17,18,19}].test.testOnly' -- 'ammonite.session.BuiltinTests.settings'`.

Reply:
Great!